### DISCUSSION TITLE
Normal to get same UMAP for RNA and SCTransform assay?

### DISCUSSION THREAD

**Discussion by dartagnan323** [satijalab/seurat#9558] [General]
Hi,
I was wondering if it was normal to get the same exact UMAP (the look is the same , just slightly different number of clusters) between the RNA and SCTransform assays.
I used same number of dimensions (30) for each and 0.8 resolution for the UMAP on RNA assay.

(seurat 5.1, in Rstudio, latest R version, mouse tissue in 4 different conditions merged in a single object, 10x 3' tech)
thank you for your help,
best regards

**Comment by rharao** [comment]
Hi, could you post images of the UMAP embeddings based on each assay, as well as the code you used to calculate them?
